handbook/tools/social-media.md

```diff
@@ -8,26 +8,78 @@ Any Launch Pad original content could make good social media posts to help us sh
+1. [Create a Medium article](#medium-articles) (finish 5 days before start date)
+2. Split the article into [mini-posts](#mini-posts) (start after creating the Medium article, finish 1 day before start date)
+3. [Write a newsletter issue](#newsletter-issues) (start after creating the Medium article, finish 3 days before start date)
+
+#### Medium Articles
+
+Articles should start off as a Google Document in the Launch Pad drive, where the author can collaborate with members of Launch Pad to write a draft. Once the draft is finalized, the author should create a [draft Medium post](https://help.medium.com/hc/en-us/articles/214874698-Your-drafts-posts) under their own account, where final copy edits should take place. Just reach out to get your Medium account added as an editor, and we will be able to submit drafts under our [shared publication](https://ubclaunchpad.com/medium).
+
+Each article should have a footer section copied from our other posts with links to our social media - for example, see the end of [this Medium article](https://medium.com/ubc-launch-pad-software-engineering-blog/what-is-ubc-launch-pad-d3bbfe6322dc).
+
+#### Mini-posts
+
+Once the article is prepared, segment it out into a series of mini-posts that can be adapted to work as Facebook and Instagram posts.
+
+Each of these should generally target a specific section of the article, with:
+
+* A brief blurb for the relevant section of the article
+* Some relevant image
+* Link to article
+  * For Facebook, the link could just be in the body of the post
+  * For Instagram, the link could be set in the profile, with a note in the post
+
+#### Newsletter Issues
+
+We currently use [Buttondown](https://buttondown.email/) as our newsletter provider. Newsletter issues have to be written in [Markdown](github.md#markdown), and should contain the following sections:
+
+* High-level summary of what Launch Pad has been up to
+* Short blurb for the article and link to it
+* Feature other content - this could be old articles, past projects, updates on current projects (like milestones, links to pull requests, etc.)
+* Brief summary of what's coming up next
+
+A good tool for drafting newsletter issues in Markdown is [hackmd.io](https://hackmd.io/) - you can log in using the UBC Launch Pad Google account.
+
+#### Short Links
+
+In lieu of a link shortener, we use [redirects (short links)](https://ubclaunchpad.com/config/#adding-a-custom-redirect) for paths on `ubclaunchpad.com` (for example, `ubclaunchpad.com/medium`). We maintain a redirect, `ubclaunchpad.com/latest`, that should always point to the latest campaign's [Medium article](#medium-articles).
+
+::: warning
+The `ubclaunchpad.com/latest` short link **should only be used in Instagram posts and for sharing directly with people** - we don't want links in longer-standing content (for example, Facebook/LinkedIn/newsletter posts) to become outdated over time!
```

Comment:
```suggestion
The `ubclaunchpad.com/latest` short link **should only be used in the Instagram profile's website field and for sharing directly with people** - we don't want links in longer-standing content (for example, Facebook/LinkedIn/newsletter posts) to become outdated over time!
```

Or something along those lines, just to clarify that they shouldn't be used in IG posts (cause then they will eventually be outdated)

Reply:
this sounds like a safer policy!